### DISCUSSION TITLE
Product Manager, Roadmapping, User testing, UX

### DISCUSSION THREAD

**Discussion by Azadi2023** [WeAreMahsaAmini/Core#489] [Introductions]
Hi all, 
I am a senior project manager and product lead with experience in Human Centered Design, Iterative Prototyping, Facilitation and discussion management, and can adapt these skills to fit the needs of different demographics and audiences. 

I believe in the context of the projects here I can be most helpful in:
- Helping keep projects on track towards the stated objectives
- Help team brainstorm and define immediate roadmaps
- Craft epics, user stories, interfaces, and experiences
- Define and run user feedback sessions and consolidate the learnings back into product roadmap

**Comment by Pink3lephant** [comment]
Hi @Azadi2023 

Nice to hear from you. We are working on this project, can you please have a look and let me know if you're interested in helping out? 

🐜  [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail.
